### DISCUSSION TITLE
When will zinc support cluster version?

### DISCUSSION THREAD

**Discussion by shuitai** [zinclabs/zinc#60] [General]

**Comment by prabhatsharma** [comment]
Looking forward to have stable APIs for now. I will soon focus on HA and distributed reads and writes as I consider them to be the most important things for zinc.

No timeline as of now.

**Reply by collimarco** [reply]
Horizontal scaling would be really interesting and a must in order to compete with ElasticSearch / OpenSearch. I was going to open an issue about horizontal scaling and then I found this discussion.
